static/js/search.js

```diff
@@ -45,6 +45,8 @@
     }
 
     window.renderBingSearchResults = () => {
+      let note = 'The search results on this page are currently using Bing. Please note that Bing search may not be as effective as Google search at indexing websites where different query strings might be used, and the results may vary from your usual experience.';
```

Comment:
Can we [localize](https://gohugo.io/content-management/multilingual/#translation-of-strings) this string? I'm not sure how - maybe we can embed the string to use as a hidden / metadata element inside the `<head>` of the search page.